week01/KthLargestElementinanArray.cpp

```diff
@@ -0,0 +1,21 @@
+// TC: O(N + 20000) Here, N = size of nums vector
+// MC: O(M) Here, M = total possible range of values [-10000, 10000] --> 20000
+
+class Solution {
+public:
+    int findKthLargest(vector<int>& nums, int k) {
+        int freq[20004];
```

Comment:
https://stackoverflow.com/questions/24136120/what-is-wrong-with-using-arrays-dynamically-allocated-in-c